### PR TITLE
Added BaseMultiRange story

### PR DESCRIPTION
**Pull request by divyanshu-rawat**
Added BaseMultiRange story.

### CODE REVIEW

**Comment by divyanshu-rawat** [comment]
@bietkul please check this as well.

File: stories/index.js [hunk 1]
```diff
@@ -74,6 +75,14 @@ storiesOf('List Components/SingleList', module)
     template: '<base-single-list :subProps="{ sortBy: \'asc\' }"/>',
   }));
 
+
+storiesOf('Range Components/MultiRange', module)
+  .add('Basic', () => ({
+    components: { BaseMultiRange },
+    template: '<base-multi-range :subProps="{ showFilter: false }"/>',
+  }))
+
```
Comment (bietkul) [inline stories/index.js]: @divyanshu-rawat Can you add more stories here

Comment (divyanshu-rawat) [reply stories/index.js]: @bietkul will do it.

Comment (divyanshu-rawat) [reply stories/index.js]: @bietkul updated PR with more stories.

Comment (divyanshu-rawat) [reply stories/index.js]: Let me send new PR netlify shows some err.

**Pull request closed without merging**